components/webpack/path-map.js

```diff
@@ -24,9 +29,9 @@ module.exports = function (genPath) {
     'chrome://resources/brave': path.join(
       genPath,
       'brave/ui/webui/resources/tsc'),
-    'chrome://resources/mojo/mojo/public': path.resolve(
-      genPath,
-      'mojo/public'),
+    ...allPrefixes('//resources/mojo', path.join(
+        genPath,
+        'ui/webui/resources/tsc/mojo')),
```

Comment:
the type definitions actually live in this folder, though compiled files may be in `mojo/public`. Its weird...